src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -351,11 +351,11 @@ func (s *Service) sendToRooms(cli *gomatrix.Client, feedURL string, feed *gofeed
 func itemToHTML(feed *gofeed.Feed, item gofeed.Item) gomatrix.HTMLMessage {
 	return gomatrix.HTMLMessage{
 		Body: fmt.Sprintf("%s: %s ( %s )",
-			html.EscapeString(feed.Title), html.EscapeString(item.Title), html.EscapeString(item.Link)),
+			html.UnescapeString(feed.Title), html.UnescapeString(item.Title), html.UnescapeString(item.Link)),
 		MsgType: "m.notice",
 		Format: "org.matrix.custom.html",
 		FormattedBody: fmt.Sprintf("<strong>%s</strong>:<br><a href=\"%s\"><strong>%s</strong></a>",
-			html.EscapeString(feed.Title), html.EscapeString(item.Link), html.EscapeString(item.Title)),
+			html.UnescapeString(feed.Title), html.UnescapeString(item.Link), html.UnescapeString(item.Title)),
```

Comment:
This allows XSS attacks if the title contains stuff like `"&#60;script&#62;stuff"`